addons/website_quote/controllers/main.py

```diff
@@ -62,23 +46,22 @@ def view(self, order_id, token=None, message=False, **post):
 
     @http.route(['/quote/accept'], type='json', auth="public", website=True)
     def accept(self, order_id, token=None, signer=None, sign=None, **post):
-        order_obj = request.registry.get('sale.order')
-        order = order_obj.browse(request.cr, SUPERUSER_ID, order_id)
-        if token != order.access_token:
+        Order = request.env['sale.order']
+        orders = Order.sudo().browse(order_id)
```

Comment:
use like order_sudo = request.env['sale.order'].sudo(). then order_sudo.browse(order_id). in whole method instance with super user is required.